hyper/cli.py

```diff
@@ -164,7 +164,12 @@ def set_request_data(args):
     body, headers, params = {}, {}, {}
     for i in args.items:
         if i.sep == SEP_HEADERS:
-            headers[i.key] = i.value
+            # :key:value case
+            if i.key == '':
```

Comment:
Generally speaking, good Python style tests for the empty string by saying `if not i.key`, rather than explicitly comparing to the empty string. For that reason, let's flip this entire block on its head, changing it to:

```python
if i.key:
    headers[i.key] = i.value
else:
    # :key:value case
    k, v = i.value.split(':', 1)
    headers[':' + k] = v
```

Let's also make this comment a little bit more expressive: something like "if the user wants to override a HTTP/2 special header there will be a leading colon, which tricks the command line parser into thinking the header is empty".